guides/common/modules/proc_obfuscating-hostnames-and-ip-addresses.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="obfuscating-hostnames-and-ip-addresses"]
+= Obfuscating hostnames and IP addresses
+
+You can use {ProjectName} to obfuscate hostnames and IP addresses when uploading host data to {Insights}. 
+Obfuscation ensures that sensitive identifying information is not exposed. 
+When obfuscation settings are enabled, {Project} obfuscates hostnames and IP addresses. 
+If obfuscation settings are enabled and the host has its own `insights-client` obfuscation settings, {Project} honors the host settings.  
+If obfuscation settings are disabled, {Project} does not obfuscate hostnames or IP addresses, regardless of client settings.
```

Comment:
Based on the PR we did, I would update this, here is how we determine if a host will get obfuscated:
```
Returns true if hostname obfuscation should be applied for a given host, based on hierarchy:
        # 1. Global setting for hostname obfuscation. - true if setting is enabled
        # 2. host fact reported by insights_client - if client reports in with an obfuscated fact
        # 3. if neither of the above, don't obfuscate.
```
```suggestion
If obfuscation settings are disabled, {Project} does not obfuscate hostnames or IP addresses, unless a client has obfuscation enabled in `insights-client`.
```